index.js

```diff
@@ -0,0 +1,57 @@
+const cron = require('node-cron')
+const Logger = require('le_node')
+const Promise = require('bluebird');
+const fs = Promise.promisifyAll(require('fs'));
+const {exec} = require('child_process');
+const formatter = require('dateformat')
+
+const CRON_TIME = process.env.CRON_TIME
+const MAX_BACKUPS = process.env.MAX_BACKUPS
+const BACKUP_DIR = "./backup"
+const DB_URL = `postgres://${process.env.DB_USER}:${process.env.DB_PASS}@${process.env.DB_HOST}:${process.env.DB_PORT}/${process.env.DB_NAME}`
+
+var logger = new Logger(
+    {
+        token: process.env.LE_TOKEN
+    });
+console.log(CRON_TIME)
+cron.schedule(CRON_TIME, async () => {
+    await clearDir()
+    await runPgDump()
+});
+
+async function runPgDump() {
+    var formattedDate = formatter(new Date(), "dd-mm-yyyy_HH_MM_ss")
+    exec(
```

Comment:
I just learned that `exec` runs the command and buffers the whole output, and once it's done, it calls the callback function, which is probably not the best idea here. There's `child_process.spawn` and it seems to stream the output, which would be preferable in our case, but I have no experience with it.

Reply:
The dump on production has 20megs already, so it'll reach 64megs in a month.